docs/products/clickhouse/concepts/columnar-databases.rst

```diff
@@ -1,8 +1,24 @@
-Columnar databases
-===================
+ClickHouse® as a columnar database
+==================================
 
-Compared to more traditional, row-oriented solutions, columnar database management systems store the data tables by columns to provide better performance and efficiency in certain applications.  As a truly columnar database, ClickHouse® also stores the values of the same column physically next to each other. This further increases the speed of retrieving the values of a column. However, it also makes it slower to retrieve complete rows, as the values of a single row are stored across different physical locations.
+Learn how columnar databases handles data and what benefits this approach has. Check out characteristics and capabilities of ClickHouse® as a columnar database.
```

Comment:
```suggestion
Learn how columnar databases handle data and what benefits this approach has. Check out the characteristics and capabilities of ClickHouse® as a columnar database.
```

Reply:
Thanks for spotting "handle" :-) Still, I'm gonna skip "the" since at this point in the article I dont mean a specific set of characteristics that the user is familiar with but in general something I'm just about to introduce.